english/2spooky/2spooky.md

```diff
@@ -6,4 +6,5 @@ mr. skeltal is nice!
 
 hahahaha
 
-[Go back to sleep.](../sleep/marshmallow.md)
+Have a laugh reading [this](./ghostbusters.md),
+or [go back to sleep](../sleep/marshmallow.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.